apps/chat-e2e/src/tests/messageTemplate.test.ts

```diff
@@ -744,19 +758,24 @@ dialTest(
 );
 
 dialTest(
-  'Message template created for the user-message with parametrized prompt',
+  'Message template created for the user-message with parametrized prompt.\n' +
+    'Replay with Message template created for the user-message with parametrized prompt',
   async ({
     dialHomePage,
     messageTemplateModal,
     messageTemplateModalAssertion,
     chatMessages,
+    conversations,
+    conversationDropdownMenu,
+    chat,
     promptData,
     sendMessage,
     dataInjector,
     variableModalDialog,
+    variableModalAssertion,
     setTestIds,
   }) => {
-    setTestIds('EPMRTC-4298');
+    setTestIds('EPMRTC-4298', 'EPMRTC-4372');
     const aVar = 'A';
```

Comment:
Can we have overlaps with the variables names? Not familiar with the functionality, so I can be wrong. Probably it should not be hardcoded, but randomized.

Reply:
it doesn't make much sense to use random variable names for this functionality